.eslintrc.js

```diff
@@ -12,6 +12,7 @@ module.exports = {
     'no-unused-vars': 'warn',
     'react/prop-types': [0],
     'react/jsx-filename-extension': [1, { extensions: ['.js', '.jsx'] }],
+    'import/prefer-default-export': 'off',
```

Comment:
Disable this in a specific file, instead of global settings. We need this rule to be enforced.